libnet/configure.ac

```diff
@@ -309,11 +331,15 @@ AC_ARG_ENABLE([samples],
 
 AM_CONDITIONAL([INSTALL_SAMPLES], [test x"$enable_samples" = xyes])
 
+AC_ARG_ENABLE([pkgconfig],
+    [AS_HELP_STRING([--enable-pkgconfig],[generate pkg-config .pc file @<:@default=no@:>@])])
+AM_CONDITIONAL([ENABLE_PKGCONFIG], [test x"$enable_pkgconfig" = xyes])
+PKG_INSTALLDIR
```

Comment:
In general, I really like shipping libnet with a pkgconfig file. It's great. I even think it should be enabled by default.